src/cookie-handler.js

```diff
@@ -0,0 +1,65 @@
+const fs = require('fs');
+const bcrypt = require('bcrypt');
+const auth = require('./auth.js');
+
+var usersCollection = JSON.parse(fs.readFileSync('./data/users.json', 'utf8'));
+const cookieName = 'logged-in';
+
+//Login
+const setCookie = function (req, res) {
+    var cookie = req.cookies[cookieName];
+    if (cookie === undefined) {
+        const login = req.body.login;
+        const password = req.body.password;
+        if (login === undefined || password === undefined) {
+            res.status(400).send('Bad request');
+        }
+
+        const [userFound] = usersCollection.filter(user => user.login === login);
+        if (!userFound) {
+            res.status(401).send('Wrong user, try /login again');
+        } else if (!checkPasswordForUser(userFound, password)) {
+            res.status(401).send('Wrong password, try /login again');
+        } else {
+            res.cookie(cookieName, login);
+            res.status(200).send('Logged in');
+        }
+    } else {
+        res.status(200).send('Already logged in, log out first');
```

Comment:
I don't care but it's not necessarily 200

Reply:
30x redirect?

Reply:
yes